src/util/system.cpp

```diff
@@ -23,15 +30,13 @@ std::string get_pid_exe(const std::string &pid)
   try {
     return std::filesystem::read_symlink(proc_path).string();
```

Comment:
You shouldn't have the exception here at all. You can use the exceptionless variant, e.g. pass an error_code (https://en.cppreference.com/w/cpp/filesystem/read_symlink.html). @jordalgo

Reply:
Ah neat. Will fix.